src/metabase/lib/remove_replace.cljc

```diff
@@ -205,10 +205,10 @@
                              (some (fn [{:keys [lib/source lib/source-uuid] :as column}]
                                      (when (and (= :source/previous-stage source) (= target-uuid source-uuid))
                                        (:lib/desired-column-alias column)))))]
-      (if target-ref-id
+      (cond-> query
         ;; We are moving to the next stage, so pass the current query as the unmodified-query-for-stage
-        (remove-local-references query stage-number query :field {} target-ref-id)
-        query))
+        target-ref-id
+        (remove-local-references stage-number query :field {} target-ref-id)))
```

Comment:
This is meant to be sugar only.

Reply:
It's about readability and making the intention clear. I guess that's what we mean by style.